src/nrepl/connectSequence.ts

```diff
@@ -325,6 +325,16 @@ function getConnectSequences(projectTypes: string[]): ReplConnectSequence[] {
   return sequences;
 }
 
+async function askToSetDefaultProjectForJackIn(project: string) {
+  const setAsDefault = await utilities.showBooleanInformationMessage(
+    `Do you want to set "${project}" as a default Jack-in project type for the current workspace?`
```

Comment:
```suggestion
    `Do you want to set "${project}" as the jack-in project type for the current workspace?`
```

I think maybe we should avoid saying "default" since once it's set there's no picking another one from a menu - it's just the project type that's used.